piston

```diff
@@ -68,23 +90,24 @@ case "$SUBCOMMAND" in
             -it \
             --name $CONTAINER_NAME \
             -e PISTON_LOG_LEVEL=DEBUG \
-            -e PISTON_FLAKE_PATH=/piston/packages \
+            -e PISTON_FLAKE_PATH=/piston/src \
             -e PISTON_RUNTIME_SET=$runtime_set \
-            -v $PWD:/piston/packages \
-            -d "$IMAGE_NAME_DEV:$IMAGE_TAG"
-    ;;
+            -v "$SCRIPT_DIR":/piston/src \
+            -v $DEV_VOLUME_NAME:/nix \
```

Comment:
does this even work? Last I tried to do something like this the directory (`/nix`) would be empty

Reply:
It works since it is a named volume, not a directory that is mounted. With named volumes, the files will be copied there if the volume is empty/doesn't exist. Doing `./piston build-dev` ensures the volume is deleted.